client/app/components/about-modal/about-modal.html

```diff
@@ -0,0 +1,6 @@
+<a href="#"  ng-click="info.open()" >
+    About
+</a>
+<div pf-about-modal is-open="info.isOpen" on-close="info.onClose()" additional-info="info.additionalInfo"
```

Comment:
so this would be `is-open="vm.isOpen()"` (probs want params on this)

same applies to other instances of `info.xxx`